packages/orchestrator-ui-components/src/messages/en-US.json

```diff
@@ -154,7 +154,21 @@
             "startedBy": "Started by",
             "startedAt": "Started at",
             "status": "Status",
-            "id": "ID"
+            "id": "ID",
```

Comment:
Nice; but please keep NL in sync.

And we might want to make this less painful in merges by sorting the keys

Reply:
Updated nl-NL.json